python/lsst/sims/catUtils/mixins/ObsMetaMixin.py

```diff
@@ -0,0 +1,50 @@
+import numpy as np
+
+__all__ = ["ObsMetadataBase"]
+
+class ObsMetadataBase(object):
+    """
+    This class provides the InstanceCatalog with a way to write out parameters from its ObsMetadata object.
+    """
+    def _get_nobj(self):
+        return len(self.get_objId())
```

Comment:
should this be

return len(self.column_by_name('objId'))

It will be effectively the same thing, but a little more consistent with how getters are designed to work.